pkg/provider/gitea/gitea.go

```diff
@@ -110,6 +110,13 @@ func (v *Provider) CreateStatus(_ context.Context, _ versioned.Interface, event
 	case "skipped":
```

Comment:
do we still have this case anymore? I see you updating this case.

Reply:
Its not needed
Initially kept those changes as it is so that in future if required can use it

But it become unused code in PAC so removed and updated PR